services/wallet/service.go

```diff
@@ -359,14 +365,20 @@ func (s *Service) Stop() error {
 	s.router.Stop()
```

Comment:
it doesn't look like we need Stop functions anymore if we're passing the context to everything we start, cancelling the context should serve the same purpose.

Btw, the references listed here are nice reads: https://www.notion.so/Code-smell-2-Pass-context-to-functions-cd49904001ca4152be006124e64f5c61